express-get-json/server.js

```diff
@@ -0,0 +1,33 @@
+import express from 'express';
+const app = express();
+const port = 8080;
+const grades = {
+  12: {
+    id: 12,
+    name: 'Tim Berners-Lee',
+    course: 'HTML',
+    score: 95
+  },
+  47: {
+    id: 47,
+    name: 'Brendan Eich',
+    course: 'JavaScript',
+    score: 100
+  },
+  273: {
+    id: 273,
+    name: 'Forbes Lindsay',
+    course: 'JavaScript',
+    score: 92
+  }
+};
+app.get('/api/grades', (req, res) => {
+  for (const x in grades) {
+    console.dir(x);
+    res.json(grades);
+  }
+});
```

Comment:
@rachel-lyy Please make sure you're sending exactly one response to each request.

---

![image](https://github.com/rachel-lyy/m2-0323-code-solutions/assets/16143771/d8dd2b61-5d42-432a-85e4-c0980ebf2d70)